docker/databricks/dbr/dbr9.1/genomics-with-glow/Dockerfile

```diff
@@ -0,0 +1,57 @@
+# ===== For the runtime environment for this image we need the databricks azure setup ============== 
+
+FROM projectglow/genomics:9.1 AS builder
+
+ENV GLOW_VERSION=1.1.1
```

Comment:
1.2.1

Reply:
Shouldn't this be 1.1.2? 1.2.1 was the version upgraded for Spark 3.2.1 (vs. 1.1.2 supporting Spark 3.1.2)